### DISCUSSION TITLE
ValueError: max() iterable argument is empty

### DISCUSSION THREAD

**Discussion by Wyclif3** [Kinggerm/GetOrganelle#340] [Q&A]
Hello,
I am assembling the chloroplast genome but I cant get a complete path. I have tried to change the word size, k-mer values as well as number of rounds but it still retun an error as shown in the log file. Any help?
[Uploading get_org.log.txt…]()

**Comment by JianjunJin** [comment]
Invalid posting:
1. no valid log uploaded.
2. not using issue format.
3. The genome completeness problem should be transferred to discussion-Q&A
